prisma/schema.prisma

```diff
@@ -36,7 +36,8 @@ model Profile {
   lastName    String
   bio         String?
   githubUrl   String?
-}
+  profilePicture String?
```

Comment:
you might need an `@default(null)` or something similar - check that you're not getting any warnings in a db with existing prior rows in the profile table

Reply:
profilePicture does not seem to cause errors in db. Previous users already have their profilePicture automatically set to NULL.